Link for formiz v2

doriandel opened the pull request:
added a small link to the issue that announce formiz v2 in the hero of formiz landing page, configured the link's style and hover effect.
<img width="1426" alt="image" src="https://user-images.githubusercontent.com/45878945/165524856-7a050856-7b6d-4960-bb9b-d6027cd2ef01.png">

yoannfleurydev commented on documentation/src/pages/index.js:
```diff
@@ -170,6 +170,15 @@ function Home() {
               Demos
             </Link>
           </div>
+          <Link
+            className={classnames(
+              'button button--link button--md text-white underline transition duration-700 ease-in-out hover:no-underline hover:text-green-100',
+              styles.getStarted,
+            )}
+            to={useBaseUrl('https://github.com/ivan-dalmet/formiz/issues/116')}
+          >
+            Formiz v2 is under construction !
+          </Link>
```
1. I think that you can use a basic HTML `a` tag for this as `Link` is [made to navigate in Docusaurus](https://docusaurus.io/docs/docusaurus-core#link) and not outside.
2. I'm not sure about `useBaseUrl` as the name doesn't match what we want to do here.
3. Be careful to use `target="_blank"` to open the link in a new tab and be careful about its security.
4.. Thanks for the contribution.

doriandel replied:
Thank you for your review ! Currently working on fixing what I've done.

doriandel commented:
Did some fixes following @[yoannfleurydev](https://github.com/yoannfleurydev) review : 

1. changed the link from a `Link` component to a basic HTML `a` tag
2. Removed `useBaseUrl`
3. added `target="_blank"` but with `rel="noreferrer"` for security purpose

ivan-dalmet commented on documentation/src/pages/index.js:
```diff
@@ -170,6 +170,17 @@ function Home() {
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Formiz v2 is under construction !
```
```suggestion
            Formiz v2 is under construction!
```

doriandel replied:
Fixed ! Thank you for your review !
